runtime/Cargo.toml

```diff
@@ -46,6 +46,7 @@ regex = { workspace = true }
 serde = { workspace = true, features = ["rc"] }
 serde_derive = { workspace = true }
 serde_json = { workspace = true }
+serde_with = { workspace = true }
```

Comment:
ideally we aren't introducing dependencies in backports to what will imminently be stable branch. is this strictly necessary?